components/centraldashboard/public/components/dashboard-view.js

```diff
@@ -119,20 +102,20 @@ export class DashboardView extends PolymerElement {
      * Observer for platformInfo property
      */
     _platformInfoChanged() {
+        let platformProps = {};
         if (this.platformInfo && this.platformInfo.providerName === 'gce') {
             this.platformName = 'GCP';
             const pieces = this.platformInfo.provider.split('/');
             let gcpProject = '';
             if (pieces.length >= 3) {
                 gcpProject = pieces[2];
             }
-            this.platformLinks = GCP_LINKS.map((l) => {
-                return {
-                    text: l.text,
-                    link: l.link + gcpProject,
-                };
-            });
+            platformProps = getGCPData(gcpProject);
         }
+        Object.entries(platformProps).forEach(([key, value]) => {
```

Comment:
I think this can be simplified for readability even more by dropping the individual platformLinks, platformName, etc. properties and having the object that gets returned by the helper function in the `cloud-platform-data` module meet a certain specification. Then, the template should reference that object's properties (ie. platformDetails.links, platformDetails.name, etc.).

See the attached suggestion.
[suggested_change.patch.txt](https://github.com/kubeflow/kubeflow/files/3162541/suggested_change.patch.txt)